src/openloco/windows/townwnd.cpp

```diff
@@ -457,7 +461,8 @@ namespace openloco::ui::windows::town
                 // Draw population graph
                 uint16_t yPos1 = -town->history[i] + (self->height - 57);
                 uint16_t yPos2 = -town->history[i + 1] + (self->height - 57);
-                gfx::draw_line(clipped, xPos, yPos1, xPos + 1, yPos2, colour::get_shade(self->colours[1], 7));
+                if (i < town->history_size - 1)
+                    gfx::draw_line(clipped, xPos, yPos1, xPos + 1, yPos2, colour::get_shade(self->colours[1], 7));
```

Comment:
Is this the change you were referring to with "fixed bug causing population graph to show 0 population for Current Date"?

Reply:
Yes

Reply:
Thanks. I've added a comment and changelog entry.